src/eei.cpp

```diff
@@ -426,7 +426,9 @@ string toHex(evm_uint256be const& value) {
 
       HERA_DEBUG << "log " << hex << dataOffset << " " << length << " " << numberOfTopics << dec << "\n";
 
-      heraAssert(!(msg.flags & EVM_STATIC), "\"log\" attempted in static mode");
+      if (msg.flags & EVM_STATIC)
+        throw StaticModeViolation{};
```

Comment:
I'm not sure how beneficial it would be, but I think it may make sense keeping the function name which cause the violation.

Reply:
I can add it to the exception, but there is no place where to use it (we should be moving from cout to proper tracing).

Reply:
We have those `HERA_DEBUGGING` lines. At least add `TODO` or `FIXME` where the exception is declared as a reminder. Plus create an issue for it.